feature-libs/storefinder/components/store-finder-search/store-finder-search.component.ts

```diff
@@ -14,11 +14,19 @@ export class StoreFinderSearchComponent {
   constructor(private routingService: RoutingService) {}
 
   findStores(address: string) {
-    this.routingService.go(['store-finder/find'], { query: address });
+    this.routingService.go(['store-finder/find'], {
+      state: {
+        query: address,
+      },
+    });
```

Comment:
we should use property `queryParams` for query params, but not `state`; similar to https://github.com/SAP/spartacus/pull/12638#discussion_r648196574
```suggestion
    this.routingService.go(['store-finder/find'], {
      queryParams: {
        query: address,
      },
    });
```